src/tezos_interop/tezos_js_bridge.js

```diff
@@ -23,6 +23,14 @@ const { inspect } = require("util");
  * @property {string} rpc_node
  * @property {number} confirmation
  * @property {string} destination
+ * 
+ * @typedef BigMapRequest
+ * @type {object}
+ * @property {"storage"} kind
+ * @property {string} rpc_node
+ * @property {number} confirmation
+ * @property {string} destination
+ * @property {any} key
```

Comment:
@EduardoRFS, can you elaborate on why multiple keys would be better? Is the goal to minimize IPC calls in Deku?

Reply:
@EduardoRFS I think it's typed properly now but I'm not sure how to check for sure. Also I'm not sure why it would be better to support multiple keys (is the idea that there's a constant overhead when invoking the bridge?)

Reply:
@d4hines not only to minimize IPC calls, you're also spamming the Tezos node.

Reply:
it also reduces the complexity and the need for the sequence_result function as now or all fail or none fail.

Reply:
@EduardoRFS I think we'd have to spam the tezos node anyway as I'm not sure there's a way to request the keys  for multiple values in one go.